lib/operations/drop_collection.js

```diff
@@ -2,22 +2,14 @@
 
 const CommandOperation = require('./command');
 const handleCallback = require('../utils').handleCallback;
-const MongoError = require('../core').MongoError;
 
 class DropCollectionOperation extends CommandOperation {
```

Comment:
Where is the actual command generated here? Shouldn't this be taking in `constructor(db, collectionName, options)` and then constructing the command off of that?